examples/case_studies/reliability_and_calibrated_prediction.myst.md

```diff
@@ -953,15 +939,11 @@ az.summary(idata_informative)
 ```
 
 ```{code-cell} ipython3
-joint_draws = az.extract_dataset(idata, group="posterior", num_samples=1000)[
-    ["alpha", "beta"]
-].to_dataframe()
+joint_draws = az.extract(idata, num_samples=1000)[["alpha", "beta"]]
```

Comment:
The fact that was a dataframe wasn't being used anywhere, so I changed that because now below we take advantage of the fact `joint_draws` is an xarray dataset.

Reply:
That's fair.